features/steps/wishlist_steps.py

```diff
@@ -68,6 +76,8 @@ def step_impl(context):
 def step_impl(context):
     """ Delete all Wishlists and load new ones """
     headers = {'Content-Type': 'application/json'}
+    # context.resp = requests.delete(context.base_url + '/wishlists/reset', headers=headers)
```

Comment:
Can you remove this commented code?

Reply:
It is there since I will try to make it work. I added this PR so you could work, I can undo the PR if you do not want this commented code.